### ISSUE TITLE
About and Place section pages

### ISSUE THREAD

**Issue by cchristopherhowardd** [aarome/aarome#293]
A handful of changes to section pages for About (https://staging.aarome.org/about) and Place (https://staging.aarome.org/about/place), based on a conversation with my colleagues Anne Coulson and Marques McClary. 

I list the first five things here and will revisit the sections after I return from an afternoon walk around the neighborhood.

1. Change the name of Place to History. 

The URL https://staging.aarome.org/about/place becomes https://staging.aarome.org/about/history, and https://staging.aarome.org/it/informazioni/luoghi becomes https://staging.aarome.org/it/informazioni/luoghi  becomes https://staging.aarome.org/it/informazioni/storia

2. On the Place index page (to become History/Storia), position Academy first and Villa Aurelia second.

3. Move Directions (https://staging.aarome.org/about/place/directions) from the Place index page to the second-to-last position on the About page (before Tours).

4. Then, move Rome Sustainable Food Project to after Tours. (I'm toying with the idea of moving RSFP from About to History but haven't made a decision yet--this page is kinda sticking out at the moment.

5. Make the appropriate changes to the website footer to reflect the changes above.

**Comment by jasloe** [comment]
No problem. Working on getting #254 at the moment but have pinned the issue and will resolve when that's pushed to code.

**Comment by cchristopherhowardd** [comment]
No hurries.

**Comment by jasloe** [comment]
-[ ] Change the name of Place to History. 

The URL https://staging.aarome.org/about/place becomes https://staging.aarome.org/about/history, and https://staging.aarome.org/it/informazioni/luoghi becomes https://staging.aarome.org/it/informazioni/luoghi  becomes https://staging.aarome.org/it/informazioni/storia

- [ ] On the Place index page (to become History/Storia), position Academy first and Villa Aurelia second.

3. Move Directions (https://staging.aarome.org/about/place/directions) from the Place index page to the second-to-last position on the About page (before Tours).

4. Then, move Rome Sustainable Food Project to after Tours. (I'm toying with the idea of moving RSFP from About to History but haven't made a decision yet--this page is kinda sticking out at the moment.

5. Make the appropriate changes to the website footer to reflect the changes above.

**Comment by jasloe** [comment]
@cchristopherhowardd –– I think that's everything. Have a look and test when you have a momemnt.

**Comment by cchristopherhowardd** [comment]
Looks good.

The link to History at https://staging.aarome.org/about is broken. It's still going to https://staging.aarome.org/about/place when it should go to https://staging.aarome.org/about/history

I changed my mind about the order of boxes on the About section page. Can you please make the last three like this?

Rome Sustainable Food Project
Tours
Directions

**Comment by cchristopherhowardd** [comment]
The link to History at https://staging.aarome.org/about is broken. It's still going to https://staging.aarome.org/about/place when it should go to https://staging.aarome.org/about/history. Same issue with the Italian pages.

**Comment by cchristopherhowardd** [comment]
It was decided yesterday to remove the Directions page ( https://staging.aarome.org/about/directions ) from the About section ( https://staging.aarome.org/about ).

After the Directions page is removed from about, it can be unpublished. The information on the page is old. I like the map image and may use it somewhere else.

(There was a decision yesterday to move Society of Fellows from the People section to About, but I'll open another ticket for this.)

**Comment by jasloe** [comment]
/about/history –– I rewrote the hack for sub-index pages described at #261 to support the new Academic Programs feature and I broke something. Fixed, now. 

I've unpublished the /about/directions node and removed it from the menu and index page. Since it's useful information, if not entirely up-to-date, the node is still reachable to Academy users in the event someone needs to refer to that information. 

> The link to History at https://staging.aarome.org/about is broken. It's still going to https://staging.aarome.org/about/place when it should go to https://staging.aarome.org/about/history. Same issue with the Italian pages.
